### PR TITLE
FEAT: launch UI of audio model.

### PR DESCRIPTION
**Pull request by hainaweiben**
In fact, I found that most audio models are not available on ModelScope.

### CODE REVIEW

File: xinference/model/audio/model_spec_modelscope.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "model_name": "whisper-small.en",
+        "model_family": "whisper",
+        "model_id": "iic/speech_whisper-small_asr_english",
```
Comment (qinxuye) [inline xinference/model/audio/model_spec_modelscope.json:5]: Is this model identical to whisper-small.en? It looks not.

Comment (hainaweiben) [reply xinference/model/audio/model_spec_modelscope.json:5]: Actually, I'm not very sure about this whisper-small.en from modelscope as shown in the picture.
image
If not, I will delete the corresponding JSON content.

Comment (hainaweiben) [reply xinference/model/audio/model_spec_modelscope.json:5]: https://modelscope.cn/models/iic/speech_whisper-small_asr_english/summary

Comment (qinxuye) [reply xinference/model/audio/model_spec_modelscope.json:5]: You can see the description, it's different I think.

Comment (hainaweiben) [reply xinference/model/audio/model_spec_modelscope.json:5]: I agree with your opinion and will delete the JSON file of this model.

File: xinference/model/audio/model_spec_modelscope.json
Comment (qinxuye) [inline xinference/model/audio/model_spec_modelscope.json:1]: Is only this file enough? Can the audio model choose to download from modelscope now?

**Comment by qinxuye** [comment]
@hainaweiben I think you can still add the support for audio models on the web UI, without considering the modelscope part.

**Review by ChengjieLi28 — APPROVED** [review]

**Pull request merged**